src/js/components/recipientLanding/table/RecipientLandingTable.jsx

```diff
@@ -32,6 +32,9 @@ const RecipientLandingTable = (props) => {
                 type={row.recipientLevel}
                 name={row.name}
                 searchString={props.searchString} />
+            <td className="recipient-list__body-cell recipient-list__body-cell_left">
+                {row.duns}
+            </td>
```

Comment:
This column re-added, but header is left out; I think this should be inserted at line 80 below:

                        <th className="recipient-list__head-cell">
                            <div className="header-cell ">
                                <div className="header-cell__text">
                                    <div className="header-cell__title header-cell__title_cap">
                                        Duns
                                    </div>
                                </div>
                                <StateLandingTableSorter
                                    field="duns"
                                    label="duns"
                                    active={{ field: props.order.field, direction: props.order.direction }}
                                    setSort={props.setSort} />
                            </div>
                        </th>